### PR TITLE
Changes for adding premium org details

### PR DESCRIPTION
**Pull request by sumesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/1311

*Description of changes:*
- Changes for adding premium org details

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-api/migrations/versions/0ab62b841b88_add_drawdown_payment_type.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    org_type_table = table('org_type',
+                           column('code', String),
+                           column('desc', String),
+                           column('default', Boolean)
+                           )
+
+    op.bulk_insert(
+        org_type_table,
+        [
+            {'code': 'PREMIUM', 'desc': 'Premium accounts', 'default': False}
```
Comment (sutherlanda) [inline auth-api/migrations/versions/0ab62b841b88_add_drawdown_payment_type.py:45]: Do we want to have one type to replace IMPLICIT/EXPLICIT now? Like STANDARD?

Comment (sutherlanda) [reply auth-api/migrations/versions/0ab62b841b88_add_drawdown_payment_type.py:45]: or BASIC

Comment (sumesh-aot) [reply auth-api/migrations/versions/0ab62b841b88_add_drawdown_payment_type.py:45]: Thats a good idea. We can stick to BASIC and PREMIUM. 👍 
Is it fine to make that change as part of next ticket (Edit premium account) ? We will need to see if it's used anywhere in UI or any other place.

Comment (sutherlanda) [reply auth-api/migrations/versions/0ab62b841b88_add_drawdown_payment_type.py:45]: Yep, sounds good to me! 👍

**Pull request merged**